demo/index.html

```diff
@@ -6,16 +6,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
     <link rel="preconnect" href="https://fonts.gstatic.com" />
-    <link rel="icon" type="image/png" sizes="32x32" href="static/favicon-32x32.png" />
-    <link rel="icon" type="image/png" sizes="16x16" href="static/favicon-16x16.png" />
```

Comment:
Why did you remove these two links?

Reply:
I had removed because of some errors, but I'm putting it back again on its place.